### PR TITLE
updated history information to oxygen

### PR DESCRIPTION
**Pull request by zhangyongND**
We added history information of oxygen.

### CODE REVIEW

**Comment by jamesluo274** [comment]
like from a history book?

**Comment by hwang1992** [comment]
Looks cool!

File: elements/oxygen.md [hunk 1]
```diff
@@ -8,5 +8,7 @@ Atomic Weight: 15.999
 
 An element that is essential to our everyday life. According to encyclopedia Britannica, Oxygen (O), nonmetallic chemical element of Group 16 (VIa, or the oxygen group) of the periodic table. Oxygen is a colourless, odourless, tasteless gas essential to living organisms, being taken up by animals, which convert it to carbon dioxide; plants, in turn, utilize carbon dioxide as a source of carbon and return the oxygen to the atmosphere. Oxygen forms compounds by reaction with practically any other element, as well as by reactions that displace elements from their combinations with each other; in many cases, these processes are accompanied by the evolution of heat and light and in such cases are called combustions. Its most important compound is water.
 
-
+Discovered date: 1774
```
Comment (emarinri) [inline elements/oxygen.md:11]: This was a very clever trick!

**Review by emarinri — APPROVED** [review]

File: elements/oxygen.md [hunk 2]
```diff
@@ -8,5 +8,7 @@ Atomic Weight: 15.999
...
+Discovered by: Joseph Priestley in Wiltshire, England and independently by Carl Wilhelm Scheele in Uppsala, Sweden
```
Comment (ernie2k) [inline elements/oxygen.md:12]: Nice!

**Pull request merged**